dev/django.dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /code
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Download Poetry into Path
-RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/poetry python3 -
+RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/opt/poetry | POETRY_VERSION=1.3.0 python3 -
```

Comment:
I'm okay with testing your hypothesis that the poetry version needs to be downgraded, but there may be a problem with the syntax

The CI checks shown at the bottom of PR are setup to prevent merging if any of them are failing

![image](https://github.com/hackforla/CivicTechJobs/assets/73561520/cbc77bc1-a71d-42f7-9613-b223674f7a22)


The linter thinks there's a problem with `POETRY_HOME` and `POETRY_VERSION` 


https://github.com/hackforla/CivicTechJobs/actions/runs/6570541381/job/17848134099?pr=458#step:7:125